src/fty-alert-engine-log.cfg

```diff
@@ -20,8 +20,8 @@ log4cplus.logger.alerts-engine-audit=DEBUG, alerts-engine-audit-log, syslog, con
 
 log4cplus.appender.alerts-engine-audit-log=log4cplus::RollingFileAppender
 log4cplus.appender.alerts-engine-audit-log.File=/var/log/app-audit/alarms-audit.log
-log4cplus.appender.alerts-engine-audit-log.MaxFileSize=16MB
-log4cplus.appender.alerts-engine-audit-log.MaxBackupIndex=3
+log4cplus.appender.alerts-engine-audit-log.MaxFileSize=64MB
+log4cplus.appender.alerts-engine-audit-log.MaxBackupIndex=10
 log4cplus.appender.alerts-engine-audit-log.Threshold=INFO
 log4cplus.appender.alerts-engine-audit-log.layout=log4cplus::PatternLayout
 log4cplus.appender.alerts-engine-audit-log.layout.ConversionPattern=%D{%Y-%m-%d %H:%M:%S} %c [%-5p] %F:%L %m%n
```

Comment:
%F:%L this should not be probably in audit file.

Reply:
Ok take into account

Reply:
Ok take into account

Reply:
Take into account